### DISCUSSION TITLE
Make traceTimestampTable be configurable.

### DISCUSSION THREAD

**Discussion by fusiyuan2010** [grafana/clickhouse-datasource#753] [General]
Am I missing something? While I can specify the trace table in the datasource configuration, I can not change the value of traceTimestampTable, while is a necessary table to look into trace details, the default value of it is 'otel_traces_trace_id_ts' and I had to create table in such name.

**Comment by SpencerTorres** [comment]
This has been reported in https://github.com/grafana/clickhouse-datasource/issues/761

Here's the explanation for the fix I added: https://github.com/grafana/clickhouse-datasource/issues/761#issuecomment-2000154659

Add your comments there if this works for you or if you need further flexibility. There's some other workarounds in that thread too.
Thanks!
